comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -134,7 +135,7 @@ func newRemoteConfigClient(deps dependencies) (rcclient.Component, error) {
 		isSystemProbe:     deps.Params.IsSystemProbe,
 	}
 
-	if pkgconfigsetup.IsRemoteConfigEnabled(pkgconfigsetup.Datadog()) {
+	if configUtils.IsRemoteConfigEnabled(pkgconfigsetup.Datadog()) {
```

Comment:
Same here, use the config comp

```suggestion
	if configUtils.IsRemoteConfigEnabled(deps.Config) {
```